opencti-platform/opencti-front/src/utils/Localization.js

```diff
@@ -1994,9 +1994,11 @@ const i18n = {
       Reset: 'Reiniciar',
       'Do you want to clear the works of this connector?':
         '¿Quieres eliminar todas las tareas de este conector?',
+      'strongly-disagree': 'muy en desacuerdo',
```

Comment:
these translations are not seen in the UI (in 'update an opinion, in the Opinion Radar, and in the vocab lists...